etc/rules/msauth_rules.xml

```diff
@@ -876,7 +876,13 @@
     <description>User Logoff Exchange.</description>
   </rule>
   
-  
+    <rule id="18262" level="3">
+    <if_sid>18107</if_sid>
+    <id>^4624$</id>
+    <match>Logon Type:   2</match>
```

Comment:
I would think this would be better if you matched with something like:

```
<match>Logon Type:\W+2 </match>
                  ^   ^-notice the space
                   \--- Anything other then A-Z, a-z, 0-9, '-', '@'
```

The reason is that this will match 2, 211, 21 just about anything starting with a 2